src/RestEase/Implementation/RawQueryParameterInfoExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace RestEase.Implementation
+{
+    /// <summary>
+    /// Extensions for raw query parameter info
+    /// </summary>
+    public static class RawQueryParameterInfoExtensions
+    {
+        /// <summary>
+        /// Serializes the raw query parameters to a query string
+        /// </summary>
+        /// <param name="rawQueryParameter"></param>
+        /// <param name="formatProvider"></param>
+        /// <returns></returns>
+        public static string SerializeToString(this IEnumerable<RawQueryParameterInfo> rawQueryParameter, IFormatProvider formatProvider)
+        {
+            if (rawQueryParameter == null || !rawQueryParameter.Any())
+                return string.Empty;
+
+            IEnumerable<string> rawQueryStrings =
+                rawQueryParameter.Select(rqp => rqp.SerializeToString(formatProvider) ?? string.Empty);
+
+            return string.Join("",rawQueryStrings);
```

Comment:
This is wrong, anyway: we add in `?` and `&` between different query strings as appropriate in `Requester`, and document that the user shouldn't append or prepend their query string with `?` or `&`.